native/cocos/bindings/manual/jsb_scene_manual.cpp

```diff
@@ -709,9 +709,9 @@ static bool js_Model_setInstancedAttribute(se::State &s) // NOLINT(readability-i
                 switch (type) {
                     case se::Object::TypedArrayType::FLOAT32: {
                         uint8_t *data = nullptr;
-                        size_t byteLength = 0;
-                        if (val.toObject()->getTypedArrayData(&data, &byteLength) && data != nullptr && byteLength > 0) {
-                            cobj->setInstancedAttribute(name, reinterpret_cast<const float *>(data), static_cast<uint32_t>(byteLength));
+                        size_t byteCount = 0;
```

Comment:
I think `byteCount` means `elementCount`？
Becasue `byteCount` is no different from `byteLength` by name.

Reply:
getTypedArrayData returns byte length of the internal buffer. 
Why change the variable name from `byteLength` to `byteCount`?